app/src/test/java/com/github/alfabravo2013/readyforexams/domain/create/AddTaskUseCaseTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.github.alfabravo2013.readyforexams.domain.create
+
+import com.github.alfabravo2013.readyforexams.domain.home.ChecklistRepository
+import com.github.alfabravo2013.readyforexams.util.Result
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+internal class AddTaskUseCaseTest {
+    private val checklistRepository = mockk<ChecklistRepository>()
+    private val useCase = AddTaskUseCase(checklistRepository)
+
+    @Test
+    @DisplayName("Given LocalDataSource has empty or blank editedDescription, When addTask, Then return Result.Failure")
```

Comment:
I think here you can just write Given editDescription is empty or blank, when ...
Same with all the others tests. Please re-check them.